decidim-forms/app/views/decidim/forms/questionnaires/_answer.html.erb

```diff
@@ -1,10 +1,9 @@
-<% field_id = "questionnaire_answers_#{answer_idx}" %>
+<% field_id = "questionnaire_responses_#{answer_idx}" %>
```

Comment:
One doubt; why changing from "answers" to "responses"?

Reply:
`questionnaire_responses_*` is the correct id, I merged from develop and had to change it.

Reply:
@jesusdb this may be related to this: https://github.com/decidim/decidim/pull/5819 we changed answers to responses to avoid default behavior of Rectify Forms loading answers unnecessarily and causing a heavy penalty performance regression.